maint_tools/update_niivue.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash -exf
+
+# Update version of niivue vendored with Nilearn
+# according to the version found in npm-requirements.txt
+#
+# requires npm
+
+set -x -e
+
+npm install "$(cat npm-requirements.txt)"
+
+if [ -f  "nilearn/plotting/data/js/niivue.umd.js" ]; then
+    rm nilearn/plotting/data/js/niivue.umd.js
+fi
+
+if [ -f  "node_modules/@niivue/niivue/node_modules/fflate/umd/index.js" ]; then
+    source_file="node_modules/@niivue/niivue/node_modules/fflate/umd/index.js"
+
+elif [ -f  "node_modules/@niivue/niivue/dist/niivue.umd.js" ]; then
+    source_file="node_modules/@niivue/niivue/dist/niivue.umd.js"
+fi
+
+cp "$source_file" nilearn/plotting/data/js/niivue.umd.js
```

Comment:
by the way I have no idea if this is a good way to update niivue: am just hacking things here

Reply:
I think this mechanism is fine. If you're not using a proper module bundler for frontend code, then your custom build/packaging script seems like a decent approach. However, if you do already have a node_modules folder managed with npm, then that would remove the need to use `unpkg` and the hosted niivue module. This way, the use of niivue isn't dependent on a network connection.

Reply:
> you do already have a node_modules folder managed with npm

we do not and I do not think we plan on having one in the near future

our implementation would just ship niivue with nilearn and should ideally not require a network connection to work (if users choose to embbed the niivue code in their report)